.editorconfig

```diff
@@ -10,5 +10,4 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 
 [*.md]
-indent_size = 4
```

Comment:
:eyes:

Reply:
I understand why this might have been added, but this creates an obstacle when providing code examples in READMEs and other Markdown files, so I removed this. It doesn't really belong here though, I can move it to a different PR.

Reply:
Makes sense!

Reply:
Restored in [b3eeefd](https://github.com/MetaMask/controllers/pull/831/commits/b3eeefd409e79b4d3d7111370d46f735e54fe7e1).

PR created here: https://github.com/MetaMask/controllers/pull/938